test/test_helper.rb

```diff
@@ -67,10 +67,6 @@ def assert_can(ability, *args)
   def assert_cannot(ability, *args)
     assert ability.cannot?(*args), "User can #{args.join(' ')} but should not"
   end
-
-  teardown do
-    User.current = nil
-  end
```

Comment:
Why was this required before but not now?

Reply:
This is redundant. It is also done in `test/test_helpers/user.rb`